homework7/task1.py

```diff
@@ -26,13 +26,23 @@ def show_contact(name):
             c = name
             print(f"Name: {i.get('name')}\nPhone number: {i.get('phone number')}")
 
+    try:
+        print(i["name2"])
+    except(KeyError):
+         print("contact with this name not found")
+
```

Comment:
the task is about adding processing exceptions in the working program, for example, you can try to delete user from phone book by key:
```
def delete_contact(name):
    try:
        del phone_book[name]
    except KeyError:
        print("contact with this name not found")
```